db/migrate/20200506054220_add_reg_group_to_user_and_document.rb

```diff
@@ -0,0 +1,6 @@
+class AddRegGroupToUserAndDocument < ActiveRecord::Migration
+  def change
+    add_column :users, :rep_group, :string
```

Comment:
@timdunn22 -- this migration should not be necessary, and actually might mess things up -- `rep_group` is managed by the `acts_as_taggable_on` gem.